.fixtures.yml

```diff
@@ -5,7 +5,6 @@ fixtures:
     choria: https://github.com/choria-io/puppet-choria.git
     concat: https://github.com/puppetlabs/puppetlabs-concat.git
     cron_core: https://github.com/puppetlabs/puppetlabs-cron_core.git
-    git: https://github.com/puppetlabs/puppetlabs-git.git
```

Comment:
Can you please remove the other mcollective entries as well?

Reply:
Tried that but then all the test failed ...
Had to put them back on by one to get all the tests green :)